lib/toy_class.rb

```diff
@@ -8,7 +8,7 @@ def self.toy_new(toy_superclass = ToyObject)
 
   def initialize(toy_superclass)
     @toy_superclass = toy_superclass
-    super
+    super()
```

Comment:
The same remark from `ToyModule` applies here: arguably it’s clearer if we give the `ToyClass` constructor a “which class am I an instance of?” argument, pass it up to `super`, and provide it in `ToyClass.toy_new`. Again we avoid the need to call `self.class`, although in this case we do still need our own implementation of `.toy_new` because it’s calling a constructor that takes different arguments than its superclasses do.